CONTRIBUTING.md

```diff
@@ -61,6 +61,19 @@ Formatting conventions:
 - comments are good, TileDB uses [doxygen](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html) for class doc strings.
 - format code using [clang-format](https://clang.llvm.org/docs/ClangFormat.html)
 
+### Building with sanitizers
+
+TileDB can be built with [clang sanitizers](https://clang.llvm.org/docs/AddressSanitizer.html) enabled. To enable them, you have to bootstrap with the `--enable-sanitizer` flag, as well as specify a vcpkg triplet compatible with ASAN (that ends with `-asan`). You can view the list of supported triplets [here](https://github.com/TileDB-Inc/TileDB/tree/dev/ports/triplets).
```

Comment:
This is not the interface we talked about. The command line need to take the base triplet, the one without the suffix `-asan`. We'll construct the triplet internally.

The reason for this was to get consistent triplet names between different users and usages.

Reply:
I added a base triplet option. It is used only if a sanitizer is enabled.